app/tests/Core/documentTests.js

```diff
@@ -1388,14 +1393,36 @@ export default [
     },
   })),
 
+  // Succeeds "6.1.26 Prohibited Document Category Name"
+  ...[
+    'CSAF Base',
+    'csaf_base',
+    //    'csaf_security_incident_response',
+    //    'csaf_informational_advisory',
+    //    'csaf_security_advisory',
+    //    'csaf_vex',
```

Comment:
@domachine: FYI Those values shall return `true` when evaluated only against 6.1.26. But might return `false` in combination with other tests...